File: OOPS_LAB/Program_3/vector3d.js
class Vector3D{
    constructor(x=0,y=0,z=0)
    {
        this.x=x;
        this.y=y;
        this.z=z;
    }

    VectorMangitude()
    {
        const sumOfSquare = Math.pow(this.x,2)+Math.pow(this.y,2)+Math.pow(this.z,2);
        const ans=Math.sqrt(sumOfSquare);
        console.log(`Magnitude of Vector : ${ans}`)
    }
}

const vec1 = new Vector3D(3,4,5)
vec1.VectorMangitude()

File: OOPS_LAB/Program_3/vector3d.py
import math

class Vector3D:
    def __init__(self,x,y,z):
        self.x=x
        self.y=y
        self.z=z

    def VectorMagnitude(self):
        sumOfSquare = pow(self.x,2)+pow(self.y,2)+pow(self.z,2)
        ans = math.sqrt(sumOfSquare)
        print(f"Magnitude of Vector {ans}")
    
if __name__=="__main__":
    vec1 = Vector3D(3,4,5)
    vec1.VectorMagnitude()